ibm/service/power/ibm_pi_constants.go

```diff
@@ -29,8 +29,10 @@ const (
 	Arg_VolumeGroupID                       = "pi_volume_group_id"
 	Arg_VolumeID                            = "pi_volume_id"
 	Arg_VolumeOnboardingID                  = "pi_volume_onboarding_id"
+	Arg_HostgroupID                         = "pi_hostgroup_id"
 
 	// Attributes
+	Attr_Hosts                                       = "hosts"
```

Comment:
Can you resort the new constants with the rest of the constants?

Reply:
Done